app/styles/app/modules/navigation.scss

```diff
@@ -239,10 +239,10 @@ $nav-line-height: 35px;
   }
 }
 
-ul #profile-menu-link:hover > ul,
-ul #profile-menu-link:focus-within > ul,
-ul #profile-menu-link ul:hover {
-  display: block;
+#profile-menu-link:hover {
+  ul {
+    display: block;
+  } 
```

Comment:
This seems like a sensible reorganization of these selectors. I'm guessing it was `:focus-within` that was breaking things for IE? Unfortunately, removing it seems to have broken keyboard navigation established in https://github.com/travis-ci/travis-web/pull/1787. I would prefer to have this working for both IE and keyboard navigation. I bet that adding a separate selector below this one for `:focus-within` would do the trick. So maybe something like:

```
#profile-menu-link {
  &:hover ul {
    display: block;
  }
  &:focus-within ul {
    display: block;
  }
}
```

Reply:
focus-within is a pseudo class and it doesnt work on ie
https://caniuse.com/#search=focus-within

Reply:
Yeah I'm assuming that it won't work in IE/edge. But if we have it as a separate selector below, then at least it'll work for chrome/FF/safari. If you want you could wrap it in a `@supports` feature query, but I think it should be unnecessary, as browsers that don't understand it would ignore it. Does that make sense?

Reply:
IE/EDGE does not ignore it - style isn't correctly handled (is skipped) . I'll try to find some good solution

Reply:
@sorta you have right, thanks1

Reply:
In 2nd commit I removed duplicated condition for  auth.signedIn

Reply:
Ahh ok cool, thanks! I'll take a look at this now.